README.md

```diff
@@ -74,9 +74,9 @@ fortio.exe server
 ```
 (at the prompt, allow the windows firewall to let connections in)
 
-Once `fortio server` is running, you can visit its web UI at [http://localhost:8080/fortio/](http://localhost:8080/fortio/)
+Once `fortio server` is running, you can visit its web UI at [http://localhost:8080/fortio/](http://localhost:8080/fortio/).
```

Comment:
These `.`s at the end of a sentence while technically correct, they prevent easy copy paste of the correct url, so undoing some.

```suggestion
Once `fortio server` is running, you can visit its web UI at [http://localhost:8080/fortio/](http://localhost:8080/fortio/)
```

Reply:
Makes sense 👍🏻
My bad